scripts/testnet/include/validators.sh

```diff
@@ -0,0 +1,128 @@
+source "$ELRONDTESTNETSCRIPTSDIR/include/terminal.sh"
+
+startValidators() {
+  setTerminalSession "elrond-nodes"
+  setTerminalLayout "tiled"
+  setWorkdirForNextCommands "$TESTNETDIR/node"
+  iterateOverValidators startSingleValidator
+}
+
+pauseValidators() {
+  iterateOverValidators pauseSingleValidator
+}
+
+resumeValidators() {
+  iterateOverValidators resumeSingleValidator
+}
+
+stopValidators() {
+  iterateOverValidators stopSingleValidator
+}
+
+iterateOverValidators() {
```

Comment:
almost duplicate code with iterateOverObservers

Reply:
+1

Reply:
can remain as it is

Reply:
Due to Bash specifics, these two duplicated functions are cleaner than a single deduplicated iterator (at least at my level of knowledge). A future rewrite in Python will allow for cleaner code.